style.css

```diff
@@ -0,0 +1,430 @@
+body{
+    color:#212121;
+    font-family:'lato',sans-serif;
+    font-style: italic;
+    font-weight: 400;
+    text-align: center;
+}
+
+.container{
+    display: grid;
+    grid-template-columns: 40% 60%;
+    grid-template-rows: 120% 15%;
+    grid-gap: 1%;
+    font-size: 1.6rem;
+    
+}
+
+.content{
+    display: grid;
+    column-count:2;
+    transition: .25s;
+    grid-gap: 5px;
+}
+
+.content .content--left{
+    display: grid;
+    grid-gap:1.5%;
+    grid-template-rows: 60.5% 38%;
+    grid-column-start: 1;
+    grid-column-end: 2;
+    grid-row-start: 1;
+    grid-row-end: 2;
+    
+}
+
+.content .content--right{
+    display: grid;
+    grid-template-rows: 32.5% 23.5%  42.5%;
+    grid-column-start: 2;
+    grid-column-end: -1;
+    grid-row-start: 1;
+    grid-row-end: -2;
+    grid-gap: 0.725%;
+}
+
+.content__header{
+    background-color: #e4e2e0;
+    border: 1px solid black;
+    grid-column-start: 1;
+    grid-column-end: 2; 
+    padding: 2.85rem;
+    transition: .3s;   
+    border-radius: 20% 0 0 0;
+    overflow: hidden;
+    position: relative;
+    animation: header;
+    animation-duration: 2s;
+    animation-fill-mode: forwards;
+    opacity: 0;
+}
+
+.content__personal{
+    background-color: #e4e2e0;
+    text-align: justify;
+    border: 1px solid black;
+    padding: .5rem 2rem; 
+    line-height: 150%;
+    word-spacing: .1rem;
+    transition: .3s;
+    border-radius:0 20% 0 0;
+    position: relative;
+    animation: personal;
+    animation-delay: 1s;
+    animation-duration: 3s;
+    animation-fill-mode: forwards;
+    opacity: 0;
+}
+
+.content__skills{
+    background-color: #e4e2e0;
+    border: 1px solid black;
+    grid-column-start: 2;
+    grid-column-end: -1;
+    transition: .3s;
+    position: relative;
+    animation: skills;
+    animation-delay: 1.5s;
+    animation-duration: 3s;
+    animation-fill-mode: forwards;
+    opacity: 0;
+}
+
+.content__education{
+    background-color: #e4e2e0;
+    padding: 0 2em;
+    border: 1px solid black;
+    grid-column-start: 2;
+    grid-column-end: -1;
+    transition: .3s;
+    border-radius:0 0 20% 0;
+    position: relative;
+    animation: education;
+    animation-delay: 2s;
+    animation-duration: 3s;
+    animation-fill-mode: forwards;
+    opacity: 0;
+}
+
+.content__social{
+    background-color: #e4e2e0;
+    padding: 0 1rem ;
+    border: 1px solid black;
+    font-size: 1em;
+    line-height: 2em;
+    grid-column-start: 1;
+    grid-column-end: 2;
+    border-radius:0 0 0 20% ;
+    transition: .3s;
+    position: relative;
+    animation: social;
+    animation-delay: 2.5s;
+    animation-duration: 3s;
+    animation-fill-mode: forwards;
+    opacity: 0;
+}
+
+tr,td,th, table {
```

Comment:
The element selectors could go up in the file

Reply:
Fixed.